codalab/worker/main.py

```diff
@@ -221,7 +222,7 @@ def parse_args():
     parser.add_argument(
         '--kubernetes-cert',
         type=str,
-        help='Contents of the SSL cert for the Kubernetes cluster. Only applicable if --bundle-runtime is set to kubernetes and --kubernetes-cert-path is set to /dev/null.',
+        help='Base64 encoded contents of the SSL cert for the Kubernetes cluster. Only applicable if --bundle-runtime is set to kubernetes and --kubernetes-cert-path is set to /dev/null.',
```

Comment:
@epicfaace Try -- quoting here (https://github.com/codalab/codalab-worksheets/blob/master/docker_config/compose_files/docker-compose.yml#L306C59-L306C59) to pass them as env variables